services/src/main/scala/cromwell/services/metadata/impl/deleter/DeleteMetadataConfig.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.services.metadata.impl.deleter
+
+import com.typesafe.config.Config
+import common.Checked
+import common.validation.Validation._
+import net.ceedubs.ficus.Ficus._
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.Try
+
+final case class DeleteMetadataConfig(nonSuccessInterval: FiniteDuration,
+                                      delayAfterWorkflowCompletion: FiniteDuration)
+
+object DeleteMetadataConfig {
+
+  def parseConfig(archiveMetadataConfig: Config): Checked[DeleteMetadataConfig] = {
+    val defaultNonSuccessInterval: FiniteDuration = 1 minute
```

Comment:
`non-success-interval` name not clear IMO

Reply:
How about `backoff-interval`?

Reply:
I endorse Saloni's suggestion